deploy/platform/redhat/redhat_docker_build.sh

```diff
@@ -23,6 +23,10 @@ do_createrepo() {
   fi
   : && createrepo -q $update_args -o ${tmpdir} ${repodir}/${FLAVOR}/RPMS
   checkstatus abort "Failure: Problem creating rpm repository in ${repodir}!" $?
+  rm -f ${tmpdir}/repodata/repomd.xml.asc
+  export GNUPGHOME=/sign_keys/.gnupg
+  gpg --local-user MDSplus --detach-sign --armor ${tmpdir}/repodata/repomd.xml
```

Comment:
Note that there is one key that has two names: `MDSplus` and `RPM-GPG-KEY-MDSplus`.

- `MDSplus` is on the build system's keyring, and thus using it to sign the RPM metadata allows the build to pass.
- `RPM-GPG-KEY-MDSplus` is not on the keyring and thus signing fails (which causes the build to fail the "release" step).   Although the name is different, this is the same key as the "public" `MDSplus` key.  (Confirmed that by importing the `RPM-GPG-KEY-MDSplus` ASCII key-file into GPG, and then doing `--list-keys`.)
- The source repo has an older (obsolete) `RPM-GPG-KEY-MDSplus` in `deploy/platform/redhat/RPM-GPG-KEY-MDSplus`.